source/ConditionSet.cpp

```diff
@@ -330,10 +341,16 @@ bool ConditionSet::IsValid() const
 
 
 
-// Check if the given condition values satisfy this set of conditions.
-bool ConditionSet::Test(const ConditionsStore &conditions) const
+bool ConditionSet::Test() const
 {
-	return Evaluate(conditions);
+	// An empty condition set was likely never loaded, and so won't have a pointer
+	// to the player's conditions. Return true in this case instead of throwing an
+	// exception.
+	if(IsEmpty())
+		return true;
+	if(!conditions)
+		throw runtime_error("Unable to Test ConditionSet without a pointer to a ConditionsStore!");
+	return Evaluate(*conditions);
```

Comment:
Why not also remove the conditions parameter from Evaluate? (It can get it directly from the class-variable if it needs it.)